frontend/src/menu-bar.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import HelpIcon from "@material-ui/icons/Help";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  title: {
+    flexGrow: 1
+  }
+}));
+
+// Create menu bar component with title and help icon (with About and FAQ links)
+export default function MenuAppBar() {
+  const classes = useStyles();
+  const [anchorElement, setAnchorElement] = React.useState(null);
+  const open = Boolean(anchorElement);
```

Comment:
Probably a react question but why does this need to be wrapped in a Boolean? Could it be used directly, or is this used for a truthy --> boolean conversion?

Reply:
I realized I misinterpreted our original UI design for the menubar (was not supposed to have two options, just a click that redirects it to a FAQ page) so I got rid of this code, but this line could be use directly without the Boolean wrapper.